### DISCUSSION TITLE
License of "scrap" library

### DISCUSSION THREAD

**Discussion by tana** [rustdesk/rustdesk#9788] [Q&A] [answered]
I have a question about license terms of the [RustDesk's version of `scrap` library](https://github.com/rustdesk/rustdesk/tree/master/libs/scrap), because I am planning to use it for software I am developing.

I assume your version of `scrap` contains both parts from original `scrap` and parts developed in RustDesk project.
Is it still licensed in MIT License (as its [Cargo.toml](https://github.com/rustdesk/rustdesk/blob/master/libs/scrap/Cargo.toml) says), or in AGPL like the whole RustDesk?

**Comment by rustdesk** [comment, marked as answer]
All code here are AGPL
